pkg/defaults/policies/files/deployment_has_ingress_network_policy.json

```diff
@@ -141,6 +141,42 @@
       ]
     }
   ],
+  "mitreAttackVectors": [
+    {
+      "tactic": "TA0001",
+      "techniques": [
+        "T1133",
+        "T1199"
+      ]
+    },
+    {
+      "tactic": "TA0005",
```

Comment:
How does not having ingress policy indicate that an adversary is trying to evade defenses?